### PR TITLE
Password Reset

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- Redirect password reset to frontend

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: twoopstracker/authentication/urls.py [hunk 1]
```diff
@@ -4,9 +4,14 @@
 from django.urls import path
 from django.views.generic.base import RedirectView
 
-from .views import GoogleLogin
+from .views import CusstomPasswordResetConfirmView, GoogleLogin
```
Comment (esirK) [inline twoopstracker/authentication/urls.py:7]: `CustomPasswordResetConfirmView`

**Review by esirK — APPROVED** [review]
Other than the small typo.
LGTM

File: twoopstracker/authentication/views.py [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 class GoogleLogin(SocialLoginView):
     adapter_class = GoogleSubOAuth2Adapter
     client_class = OAuth2Client
+
+
+class CustomPasswordResetConfirmView(PasswordResetConfirmView):
+    def get(self, request, *args, **kwargs):
+        uid = kwargs["uid"]
+        token = kwargs["token"]
+        url = f"{settings.TWOOPSTRACKER_CONFIRM_RESET_PASSWORD_URL}/{uid}/{token}"
```
Comment (kilemensi) [inline twoopstracker/authentication/views.py:19]: If `TWOOPSTRACKER_CONFIRM_RESET_PASSWORD_URL` ends with `/`, wouldn't this cause an issue?

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**